install/kubernetes/templates/istio-mixer-configs.yaml.tmpl

```diff
@@ -0,0 +1,408 @@
+apiVersion: "config.istio.io/v1alpha2"
```

Comment:
what is the different between this file and istio-mixer.yaml.tmpl

Reply:
This file contains the config objects (custom resources) for mixer, while istio-mixer.yaml.tmpl specifies the mixer deployment (i.e. Service, ServiceAccount, Deployment, etc).

CRDs were in between them. I just split the files into two so that the script can invoke `mixs crd all` in between.